mac

```diff
@@ -191,7 +197,17 @@ if ! [ -d "$HOME/.asdf/plugins/elixir" ]; then
   asdf global elixir 1.5.1
 fi
 
+# Install NodeJS
+
+if ! [ -d "$HOME/.asdf/plugins/nodejs" ]; then
+  asdf plugin-add nodejs https://github.com/asdf-vm/asdf-nodejs.git
+  asdf install nodejs 7.4.0
+  asdf global elixir 7.4.0
```

Comment:
@bokay is this correct `asdf global elixir 7.4.0` version number seems of to me :-)

Reply:
Ouch! Indeed!